src/main/java/com/sap/sgs/phosphor/fosstars/model/Confidence.java

```diff
@@ -66,28 +69,60 @@ static double adjust(double value) {
   }
 
   /**
-   * Calculates a confidence level based on a number of unknown feature values.
+   * Calculates a confidence level based on a number of values.
+   * The methods takes into account the following:
+   * <ul>
+   *   <li>number of unknown values</li>
+   *   <li>score values with their confidences and weights</li>
+   *   <li>values with confidences</li>
+   * </ul>
    *
    * @param values The features values.
    * @return The confidence level.
    */
-  static double make(Value... values) {
+  static double make(List<Value> values) {
```

Comment:
We use `SetValue` almost every place. Does it make sense to have a common representation everywhere?

Reply:
Do you mean `Set<Value>` or something else? Once we need `make(Set<Value>)` method, we can definitely add it. Currently, the `make()` method is used with a `List` returned by `ScoreValue.usedValues()`.